lib/sfn/config.rb

```diff
@@ -42,7 +42,7 @@ class Config < Bogo::Config
           v
         end
       },
-      :description => 'Provider credentials'
+      :description => 'Provider credentials (CredentialName:CredentialValue)'
```

Comment:
telling users the separation character would be helpful:
`Provider credentials (CredentialName:CredentialValue[,...])`